packages/firestore/src/util/log.ts

```diff
@@ -19,37 +19,42 @@
 import { SDK_VERSION } from '../core/version';
 import { AnyJs } from './misc';
 import { PlatformSupport } from '../platform/platform';
+import { Logger, LogLevel as _LogLevel } from '@firebase/logger';
+
+const client = new Logger();
 
 export enum LogLevel {
   DEBUG,
   ERROR,
   SILENT
 }
 
-let logLevel = LogLevel.ERROR;
-
 // Helper methods are needed because variables can't be exported as read/write
 export function getLogLevel(): LogLevel {
-  return logLevel;
+  if (client.logLevel === _LogLevel.DEBUG) {
+    return LogLevel.DEBUG;
+  }
+
+  if (client.logLevel === _LogLevel.SILENT) {
```

Comment:
How about else if or a switch statement?

Reply:
I'll refactor this to an `if-else-if-else`, I like to use early returns.